server/swagger.json

```diff
@@ -4080,6 +4087,22 @@
 				}
 			}
 		},
+    "DashboardRange": {
+      "type": "object",
+      "description": "A description of a particular axis for a visualization",
+      "properties": {
+        "bounds": {
+          "type": "array",
+          "minItems": 0,
+          "maxItems": 2,
+          "description": "The extents of an axis in the form [lower, upper]. Clients determine whether bounds are to be inclusive or exclusive of their limits",
+          "items": {
+            "type": "integer",
+            "format": "int32"
```

Comment:
I don't think there is any reason to restrict this to 32-bit integers.  For example, there is more than 32bit integer memory on machines.  Should be 64-bit integers.